code-editor/scripts/on-start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -ex
+ASI_VERSION=0.3.0
+
+# PARAMETERS 
+IDLE_TIME=3600  # in seconds, change this to desired idleness time before app shuts down
+
+AUTOSTOP_FILE_URL="https://github.com/aws-samples/sagemaker-studio-apps-lifecycle-config-examples/releases/download/sagemaker_code_editor_auto_shut_down-$ASI_VERSION.tar.gz"
+
+echo "Fetching the autostop script"
+echo "Downloading the autostop script package"
+
+# Download asset from corresponding repo release
+curl -L -o sagemaker_code_editor_auto_shut_down-$ASI_VERSION.tar.gz $AUTOSTOP_FILE_URL
+
+echo "Extracting the package"
+tar -xvzf "sagemaker_code_editor_auto_shut_down-$ASI_VERSION.tar.gz"
+
+# Moving the autostop.py to the current directory
+# Adjust the path according to the structure inside the tar.gz file
+echo "Moving the autostop.py to the current directory"
+mv sagemaker_code_editor_auto_shut_down/python-package/src/sagemaker_code_editor_auto_shut_down/autostop.py ./
+
+echo "Detecting Python install with boto3 install"
+sudo apt-get update -y
+sudo apt-get install -y vim
```

Comment:
Is this required?

Reply:
Good catch, it is not required and removed + verified testing